src/V3Width.cpp

```diff
@@ -3298,14 +3298,37 @@ class WidthVisitor final : public AstNVisitor {
                     relinkHandle.relink(patp);
                 }
             }
+
+            std::vector<std::pair<AstNodeDType*, AstPatMember*>> type_defaults;
+            for (AstPatMember* patp = VN_CAST(nodep->itemsp(), PatMember); patp;
```

Comment:
```suggestion
            for (AstPatMember* nextp, * patp = VN_CAST(nodep->itemsp(), PatMember); patp; patp = nextp) {
                nextp = VN_CAST(patp->nextp(), PatMember);
```

And similar in below loop. This way there's no duplicate code twice computing nextp/patp.